types/src/state_store/state_key.rs

```diff
@@ -154,6 +154,20 @@ impl StateKey {
     pub fn get_shard_id(&self) -> u8 {
         CryptoHash::hash(self).nibble(0)
     }
+
+    pub fn is_aptos_path(&self) -> bool {
```

Comment:
nit: `is_aptos_code` or similar? so that the function name describes we process modules

Reply:
done